lib/vdc/UnstructuredGridLayered.cpp

```diff
@@ -224,15 +224,45 @@ float UnstructuredGridLayered::GetValueLinear(const CoordType &coords) const
 
     // Interpolate value inside bottom face
     //
+    float mv = GetMissingValue();
+
     float  z0 = 0.0;
     size_t k0 = indices[1];
-    for (int i = 0; i < lambda.size(); i++) { z0 += AccessIJK(nodes2D[i], k0) * lambda[i]; }
+    for (int i = 0; i < lambda.size(); i++) {
```

Comment:
I don't know what kind of information `lambda` holds, but it isn't obvious to me that lambda has a small (less than 2^31) size.  Recommend giving `i` a type of `size_t`.

Reply:
Done.